recipe/meta.yaml

```diff
@@ -1,5 +1,4 @@
-{% set version = "1.1.2" %}
-
+{% set version = '1.1.2.1' %}
```

Comment:
Sorry. I missed that you upgraded it here too and merged #3  :-/
Can you rebase this?

Reply:
Wow, bad timing! :-D I'll rebase now.

Reply:
Yeah. I saw your PR and wanted to make it so we would get the latest version too. Sorry :grimacing:

Reply:
No worries! I always update to the latest version as part of these rebuilds FYI.